Cargo.toml

```diff
@@ -290,7 +290,7 @@ async-trait = "0.1.53"
 axum = "0.5.16"
 base64 = "0.13.0"
 backtrace = "0.3.58"
-bcs = { git = "https://github.com/aptos-labs/bcs", rev = "2cde3e8446c460cb17b0c1d6bac7e27e964ac169" }
+bcs = { git = "https://github.com/zekun000/bcs.git", rev = "d31fab9d81748e2594be5cd5cdf845786a30562d" }
```

Comment:
Note: need to point this to either an updated fork or upstream if we can get it patched timely.